stdlib/tkinter/__init__.pyi

```diff
@@ -3021,7 +3024,28 @@ class Text(Widget, XView, YView):
     config = configure
     def bbox(self, index: _TextIndex) -> tuple[int, int, int, int] | None: ...  # type: ignore[override]
     def compare(self, index1: _TextIndex, op: Literal["<", "<=", "==", ">=", ">", "!="], index2: _TextIndex) -> bool: ...
-    def count(self, index1, index2, *args): ...  # TODO
+    if sys.version_info >= (3, 13):
+        @overload
+        def count(
+            self, index1: _TextIndex, index2: _TextIndex, *options: Unpack[tuple[_CountOptions]], return_ints: Literal[True]
```

Comment:
If I'm reading the docstring right, this isn't quite correct: if one of the options is `update`, then that doesn't produce an extra value.

Typing this fully precisely seems difficult and possibly not worth the trouble.

cc @Akuli for opinions on tkinter.

Reply:
This method is just awful to work with without the `return_ints` parameter. Before that parameter existed, I wrote my own just to avoid figuring out how it works.

That said, typing this method is probably worth the effort:
- it is tempting to use `.count()` and easy to forget some corner case
- most people can't assume Python 3.13 yet

I think I'll just write the overloads and push directly into this PR's branch.